src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp

```diff
@@ -684,7 +684,7 @@ struct EvolutionMetavars {
       const std::vector<std::string>& deadlocked_components) {
     gh::deadlock::run_deadlock_analysis_simple_actions<
         gh_dg_element_array, control_components, interpolation_target_tags,
-        false>(cache, deadlocked_components);
+        tmpl::list<AhA, AhB>, false>(cache, deadlocked_components);
```

Comment:
What about AhC?

Reply:
AhC can't deadlock the simulation because the evolution doesn't depend on it. Therefore I didn't include it here. Think it could be helpful to add?